vendor/k8s.io/code-generator/generate-groups.sh

```diff
@@ -62,7 +62,7 @@ done
 
 if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
   echo "Generating deepcopy funcs"
-  ${GOPATH}/bin/deepcopy-gen -v10 --input-dirs $(codegen::join , "${FQ_APIS[@]}") -O zz_generated.deepcopy --bounding-dirs ${APIS_PKG} "$@"
+  ${GOPATH}/bin/deepcopy-gen --input-dirs $(codegen::join , "${FQ_APIS[@]}") -O zz_generated.deepcopy --bounding-dirs ${APIS_PKG} "$@"
```

Comment:
we shouldn't commit this change.

Reply:
that's weird. . .

Reply:
For the record, this comes out of running `dep ensure` in my project directory. I still do get the zz_generated.deepcopy.go without running dep but am seeing a huge dump of errors:
```. . .71] findTypesIn github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1
ERROR: logging before flag.Parse: I0328 12:43:37.981025   11034 parse.go:471] findTypesIn github.com/spf13/pflag
ERROR: logging before flag.Parse: I0328 12:43:37.981050   11034 parse.go:481] findTypesIn github.com/spf13/pflag: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981059   11034 parse.go:471] findTypesIn go/ast
ERROR: logging before flag.Parse: I0328 12:43:37.981070   11034 parse.go:481] findTypesIn go/ast: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981087   11034 parse.go:471] findTypesIn go/doc
ERROR: logging before flag.Parse: I0328 12:43:37.981107   11034 parse.go:481] findTypesIn go/doc: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981120   11034 parse.go:471] findTypesIn go/parser
ERROR: logging before flag.Parse: I0328 12:43:37.981132   11034 parse.go:481] findTypesIn go/parser: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981149   11034 parse.go:471] findTypesIn go/scanner
ERROR: logging before flag.Parse: I0328 12:43:37.981167   11034 parse.go:481] findTypesIn go/scanner: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981177   11034 parse.go:471] findTypesIn go/token
ERROR: logging before flag.Parse: I0328 12:43:37.981198   11034 parse.go:481] findTypesIn go/token: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981212   11034 parse.go:471] findTypesIn golang.org/x/net/http2
ERROR: logging before flag.Parse: I0328 12:43:37.981223   11034 parse.go:481] findTypesIn golang.org/x/net/http2: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981233   11034 parse.go:471] findTypesIn golang.org/x/net/http2/hpack
ERROR: logging before flag.Parse: I0328 12:43:37.981246   11034 parse.go:481] findTypesIn golang.org/x/net/http2/hpack: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981255   11034 parse.go:471] findTypesIn golang.org/x/net/idna
ERROR: logging before flag.Parse: I0328 12:43:37.981268   11034 parse.go:481] findTypesIn golang.org/x/net/idna: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981278   11034 parse.go:471] findTypesIn golang.org/x/net/lex/httplex
ERROR: logging before flag.Parse: I0328 12:43:37.981291   11034 parse.go:481] findTypesIn golang.org/x/net/lex/httplex: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981300   11034 parse.go:471] findTypesIn golang.org/x/text/secure/bidirule
ERROR: logging before flag.Parse: I0328 12:43:37.981314   11034 parse.go:481] findTypesIn golang.org/x/text/secure/bidirule: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981323   11034 parse.go:471] findTypesIn golang.org/x/text/transform
ERROR: logging before flag.Parse: I0328 12:43:37.981336   11034 parse.go:481] findTypesIn golang.org/x/text/transform: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981345   11034 parse.go:471] findTypesIn golang.org/x/text/unicode/bidi
ERROR: logging before flag.Parse: I0328 12:43:37.981358   11034 parse.go:481] findTypesIn golang.org/x/text/unicode/bidi: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981367   11034 parse.go:471] findTypesIn golang.org/x/text/unicode/norm
ERROR: logging before flag.Parse: I0328 12:43:37.981380   11034 parse.go:481] findTypesIn golang.org/x/text/unicode/norm: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981389   11034 parse.go:471] findTypesIn gopkg.in/inf.v0
ERROR: logging before flag.Parse: I0328 12:43:37.981403   11034 parse.go:481] findTypesIn gopkg.in/inf.v0: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981412   11034 parse.go:471] findTypesIn hash
ERROR: logging before flag.Parse: I0328 12:43:37.981425   11034 parse.go:481] findTypesIn hash: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981434   11034 parse.go:471] findTypesIn hash/crc32
ERROR: logging before flag.Parse: I0328 12:43:37.981448   11034 parse.go:481] findTypesIn hash/crc32: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981457   11034 parse.go:471] findTypesIn internal/cpu
ERROR: logging before flag.Parse: I0328 12:43:37.981470   11034 parse.go:481] findTypesIn internal/cpu: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981479   11034 parse.go:471] findTypesIn internal/nettrace
ERROR: logging before flag.Parse: I0328 12:43:37.981492   11034 parse.go:481] findTypesIn internal/nettrace: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981502   11034 parse.go:471] findTypesIn internal/poll
ERROR: logging before flag.Parse: I0328 12:43:37.981515   11034 parse.go:481] findTypesIn internal/poll: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981524   11034 parse.go:471] findTypesIn internal/race
ERROR: logging before flag.Parse: I0328 12:43:37.981537   11034 parse.go:481] findTypesIn internal/race: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981546   11034 parse.go:471] findTypesIn internal/singleflight
ERROR: logging before flag.Parse: I0328 12:43:37.981559   11034 parse.go:481] findTypesIn internal/singleflight: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981569   11034 parse.go:471] findTypesIn internal/syscall/unix
ERROR: logging before flag.Parse: I0328 12:43:37.981581   11034 parse.go:481] findTypesIn internal/syscall/unix: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981591   11034 parse.go:471] findTypesIn io
ERROR: logging before flag.Parse: I0328 12:43:37.981603   11034 parse.go:481] findTypesIn io: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981613   11034 parse.go:471] findTypesIn io/ioutil
ERROR: logging before flag.Parse: I0328 12:43:37.981631   11034 parse.go:481] findTypesIn io/ioutil: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981641   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/api/resource
ERROR: logging before flag.Parse: I0328 12:43:37.981654   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/api/resource: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981663   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/apis/meta/v1
ERROR: logging before flag.Parse: I0328 12:43:37.981676   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/apis/meta/v1: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981686   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/conversion
ERROR: logging before flag.Parse: I0328 12:43:37.981699   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/conversion: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981708   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/conversion/queryparams
ERROR: logging before flag.Parse: I0328 12:43:37.981721   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/conversion/queryparams: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981731   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/fields
ERROR: logging before flag.Parse: I0328 12:43:37.981744   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/fields: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981755   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/labels
ERROR: logging before flag.Parse: I0328 12:43:37.981768   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/labels: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981777   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.981790   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/runtime: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981799   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/runtime/schema
ERROR: logging before flag.Parse: I0328 12:43:37.981812   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/runtime/schema: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981821   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/selection
ERROR: logging before flag.Parse: I0328 12:43:37.981834   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/selection: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981843   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/types
ERROR: logging before flag.Parse: I0328 12:43:37.981856   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/types: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981866   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/errors
ERROR: logging before flag.Parse: I0328 12:43:37.981879   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/errors: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981888   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/intstr
ERROR: logging before flag.Parse: I0328 12:43:37.981901   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/intstr: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981910   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/json
ERROR: logging before flag.Parse: I0328 12:43:37.981923   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/json: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981933   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/net
ERROR: logging before flag.Parse: I0328 12:43:37.981945   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/net: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981955   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.981968   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/runtime: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.981977   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/sets
ERROR: logging before flag.Parse: I0328 12:43:37.981990   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/sets: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982002   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/validation
ERROR: logging before flag.Parse: I0328 12:43:37.982015   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/validation: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982024   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/validation/field
ERROR: logging before flag.Parse: I0328 12:43:37.982037   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/validation/field: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982047   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/util/wait
ERROR: logging before flag.Parse: I0328 12:43:37.982060   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/util/wait: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982069   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/watch
ERROR: logging before flag.Parse: I0328 12:43:37.982082   11034 parse.go:481] findTypesIn k8s.io/apimachinery/pkg/watch: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982091   11034 parse.go:471] findTypesIn k8s.io/apimachinery/third_party/forked/golang/reflect
ERROR: logging before flag.Parse: I0328 12:43:37.982105   11034 parse.go:481] findTypesIn k8s.io/apimachinery/third_party/forked/golang/reflect: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982114   11034 parse.go:471] findTypesIn log
ERROR: logging before flag.Parse: I0328 12:43:37.982127   11034 parse.go:481] findTypesIn log: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982136   11034 parse.go:471] findTypesIn math
ERROR: logging before flag.Parse: I0328 12:43:37.982149   11034 parse.go:481] findTypesIn math: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982158   11034 parse.go:471] findTypesIn math/big
ERROR: logging before flag.Parse: I0328 12:43:37.982171   11034 parse.go:481] findTypesIn math/big: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982181   11034 parse.go:471] findTypesIn math/bits
ERROR: logging before flag.Parse: I0328 12:43:37.982194   11034 parse.go:481] findTypesIn math/bits: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982203   11034 parse.go:471] findTypesIn math/rand
ERROR: logging before flag.Parse: I0328 12:43:37.982216   11034 parse.go:481] findTypesIn math/rand: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982226   11034 parse.go:471] findTypesIn mime
ERROR: logging before flag.Parse: I0328 12:43:37.982239   11034 parse.go:481] findTypesIn mime: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982249   11034 parse.go:471] findTypesIn mime/multipart
ERROR: logging before flag.Parse: I0328 12:43:37.982262   11034 parse.go:481] findTypesIn mime/multipart: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982273   11034 parse.go:471] findTypesIn mime/quotedprintable
ERROR: logging before flag.Parse: I0328 12:43:37.982286   11034 parse.go:481] findTypesIn mime/quotedprintable: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982295   11034 parse.go:471] findTypesIn net
ERROR: logging before flag.Parse: I0328 12:43:37.982308   11034 parse.go:481] findTypesIn net: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982318   11034 parse.go:471] findTypesIn net/http
ERROR: logging before flag.Parse: I0328 12:43:37.982332   11034 parse.go:481] findTypesIn net/http: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982341   11034 parse.go:471] findTypesIn net/http/httptrace
ERROR: logging before flag.Parse: I0328 12:43:37.982355   11034 parse.go:481] findTypesIn net/http/httptrace: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982365   11034 parse.go:471] findTypesIn net/http/internal
ERROR: logging before flag.Parse: I0328 12:43:37.982379   11034 parse.go:481] findTypesIn net/http/internal: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982388   11034 parse.go:471] findTypesIn net/textproto
ERROR: logging before flag.Parse: I0328 12:43:37.982401   11034 parse.go:481] findTypesIn net/textproto: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982410   11034 parse.go:471] findTypesIn net/url
ERROR: logging before flag.Parse: I0328 12:43:37.982423   11034 parse.go:481] findTypesIn net/url: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982432   11034 parse.go:471] findTypesIn os
ERROR: logging before flag.Parse: I0328 12:43:37.982443   11034 parse.go:481] findTypesIn os: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982453   11034 parse.go:471] findTypesIn os/user
ERROR: logging before flag.Parse: I0328 12:43:37.982465   11034 parse.go:481] findTypesIn os/user: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982471   11034 parse.go:471] findTypesIn path
ERROR: logging before flag.Parse: I0328 12:43:37.982478   11034 parse.go:481] findTypesIn path: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982483   11034 parse.go:471] findTypesIn path/filepath
ERROR: logging before flag.Parse: I0328 12:43:37.982490   11034 parse.go:481] findTypesIn path/filepath: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982495   11034 parse.go:471] findTypesIn reflect
ERROR: logging before flag.Parse: I0328 12:43:37.982502   11034 parse.go:481] findTypesIn reflect: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982507   11034 parse.go:471] findTypesIn regexp
ERROR: logging before flag.Parse: I0328 12:43:37.982514   11034 parse.go:481] findTypesIn regexp: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982519   11034 parse.go:471] findTypesIn regexp/syntax
ERROR: logging before flag.Parse: I0328 12:43:37.982526   11034 parse.go:481] findTypesIn regexp/syntax: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982531   11034 parse.go:471] findTypesIn runtime
ERROR: logging before flag.Parse: I0328 12:43:37.982538   11034 parse.go:481] findTypesIn runtime: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982543   11034 parse.go:471] findTypesIn runtime/debug
ERROR: logging before flag.Parse: I0328 12:43:37.982550   11034 parse.go:481] findTypesIn runtime/debug: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982555   11034 parse.go:471] findTypesIn runtime/internal/atomic
ERROR: logging before flag.Parse: I0328 12:43:37.982562   11034 parse.go:481] findTypesIn runtime/internal/atomic: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982567   11034 parse.go:471] findTypesIn runtime/internal/sys
ERROR: logging before flag.Parse: I0328 12:43:37.982574   11034 parse.go:481] findTypesIn runtime/internal/sys: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982580   11034 parse.go:471] findTypesIn sort
ERROR: logging before flag.Parse: I0328 12:43:37.982587   11034 parse.go:481] findTypesIn sort: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982647   11034 parse.go:471] findTypesIn strconv
ERROR: logging before flag.Parse: I0328 12:43:37.982655   11034 parse.go:481] findTypesIn strconv: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982663   11034 parse.go:471] findTypesIn strings
ERROR: logging before flag.Parse: I0328 12:43:37.982676   11034 parse.go:481] findTypesIn strings: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982684   11034 parse.go:471] findTypesIn sync
ERROR: logging before flag.Parse: I0328 12:43:37.982694   11034 parse.go:481] findTypesIn sync: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982705   11034 parse.go:471] findTypesIn sync/atomic
ERROR: logging before flag.Parse: I0328 12:43:37.982716   11034 parse.go:481] findTypesIn sync/atomic: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982724   11034 parse.go:471] findTypesIn syscall
ERROR: logging before flag.Parse: I0328 12:43:37.982735   11034 parse.go:481] findTypesIn syscall: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982743   11034 parse.go:471] findTypesIn text/template
ERROR: logging before flag.Parse: I0328 12:43:37.982752   11034 parse.go:481] findTypesIn text/template: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982759   11034 parse.go:471] findTypesIn text/template/parse
ERROR: logging before flag.Parse: I0328 12:43:37.982784   11034 parse.go:481] findTypesIn text/template/parse: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982793   11034 parse.go:471] findTypesIn time
ERROR: logging before flag.Parse: I0328 12:43:37.982814   11034 parse.go:481] findTypesIn time: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982825   11034 parse.go:471] findTypesIn unicode
ERROR: logging before flag.Parse: I0328 12:43:37.982834   11034 parse.go:481] findTypesIn unicode: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982846   11034 parse.go:471] findTypesIn unicode/utf16
ERROR: logging before flag.Parse: I0328 12:43:37.982855   11034 parse.go:481] findTypesIn unicode/utf16: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982862   11034 parse.go:471] findTypesIn unicode/utf8
ERROR: logging before flag.Parse: I0328 12:43:37.982871   11034 parse.go:481] findTypesIn unicode/utf8: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.982878   11034 parse.go:471] findTypesIn unsafe
ERROR: logging before flag.Parse: I0328 12:43:37.982890   11034 parse.go:481] findTypesIn unsafe: package is not user requested
ERROR: logging before flag.Parse: I0328 12:43:37.983773   11034 deepcopy.go:146] Considering pkg "github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1"
ERROR: logging before flag.Parse: I0328 12:43:37.983792   11034 deepcopy.go:162]   tag.value: "package", tag.register: false
ERROR: logging before flag.Parse: I0328 12:43:37.983801   11034 deepcopy.go:187] Package "github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1" needs generation
ERROR: logging before flag.Parse: I0328 12:43:37.983814   11034 execute.go:217] Processing package "v1alpha1", disk location "/home/ltulloch/src/crd/src/github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1"
ERROR: logging before flag.Parse: I0328 12:43:37.983840   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AddToScheme is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983856   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthInfo is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983866   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderConfig is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983875   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderConfigItem is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983886   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderType is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983895   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.Cluster is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983903   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterList is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983917   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterSpec is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983926   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterStatus is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983934   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.KubernetesAPIEndpoints is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983942   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.Resource is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983951   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.SchemeBuilder is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983959   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.SchemeGroupVersion is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983967   11034 deepcopy.go:269] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ServerAddressByClientCIDR is copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983975   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.addKnownTypes is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983983   11034 deepcopy.go:266] Type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.localSchemeBuilder is not copyable
ERROR: logging before flag.Parse: I0328 12:43:37.983995   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthInfo
ERROR: logging before flag.Parse: I0328 12:43:37.984299   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderConfig
ERROR: logging before flag.Parse: I0328 12:43:37.984555   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderConfigItem
ERROR: logging before flag.Parse: I0328 12:43:37.984723   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.AuthProviderType
ERROR: logging before flag.Parse: I0328 12:43:37.984870   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.Cluster
ERROR: logging before flag.Parse: I0328 12:43:37.985178   11034 parse.go:326] importPackage k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.985190   11034 parse.go:332] importPackage k8s.io/apimachinery/pkg/runtime, canonical path is k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.985200   11034 parse.go:391] typeCheckPackage k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.985208   11034 parse.go:394] typeCheckPackage k8s.io/apimachinery/pkg/runtime already done
ERROR: logging before flag.Parse: I0328 12:43:37.985217   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.987094   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterList
ERROR: logging before flag.Parse: I0328 12:43:37.987388   11034 parse.go:326] importPackage k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.987401   11034 parse.go:332] importPackage k8s.io/apimachinery/pkg/runtime, canonical path is k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.987411   11034 parse.go:391] typeCheckPackage k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.987420   11034 parse.go:394] typeCheckPackage k8s.io/apimachinery/pkg/runtime already done
ERROR: logging before flag.Parse: I0328 12:43:37.987428   11034 parse.go:471] findTypesIn k8s.io/apimachinery/pkg/runtime
ERROR: logging before flag.Parse: I0328 12:43:37.988210   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterSpec
ERROR: logging before flag.Parse: I0328 12:43:37.988435   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ClusterStatus
ERROR: logging before flag.Parse: I0328 12:43:37.988643   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.KubernetesAPIEndpoints
ERROR: logging before flag.Parse: I0328 12:43:37.988918   11034 deepcopy.go:579] Generating deepcopy function for type github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1.ServerAddressByClientCIDR
ERROR: logging before flag.Parse: I0328 12:43:37.989080   11034 execute.go:67] Assembling file "/home/ltulloch/src/crd/src/github.com/pmorie/cluster-registry-crd/pkg/apis/clusterregistry/v1alpha1/zz_generated.deepcopy.go"
ERROR: logging before flag.Parse: I0328 12:43:37.991719   11034 main.go:83] Completed successfully.
```